samples/project-anfield/import_cluster_config.py

```diff
@@ -180,7 +183,17 @@ def create_sources(source, environment, node):
     try:
         body = RegisterProtectionSourceParameters()
         body.environment = environment
-        if environment == env_enum.KGENERICNAS:
+        if environment == env_enum.KSQL:
+            name =  node["protectionSource"]["name"]
+            body = RegisterApplicationServersParameters()
+            body.applications = [env_enum.KSQL]
+            id = node["protectionSource"]["id"]
+            body.protection_source_id = source_mapping[id]
+            resp = cohesity_client.protection_sources.create_register_application_servers(body)
+            time.sleep(30)
```

Comment:
Why sleep 30 seconds ?

Reply:
Registering as mssql source takes certain time, if we arent sleeping, job creation is failing stating source is still in registration process. Instead of sleep for 30sec we can check for source registration status till it's successful

Reply:
Yes, that way its more deterministic. poll for every 10 seconds and bail out after 1 min

Reply:
done